### PR TITLE
Header Component

### PR DESCRIPTION
**Pull request by Mazoda**

### CODE REVIEW

**Review by Halasalhab — APPROVED** [review]

File: public/index.html [hunk 1]
```diff
@@ -2,42 +2,17 @@
...
-    <!--
-      Notice the use of %PUBLIC_URL% in the tags above.
-      It will be replaced with the URL of the `public` folder during the build.
-      Only files inside the `public` folder can be referenced from the HTML.
-
-      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
     <title>React App</title>
```
Comment (Rahaf-96) [inline public/index.html]: would you change this to your app name please

File: src/Components/Header/Header.jsx [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+  MenuItem,
+} from '@mui/material';
+import NotificationsNoneIcon from '@mui/icons-material/NotificationsNone';
+import SearchIcon from '@mui/icons-material/Search';
+import MenuIcon from '@mui/icons-material/Menu';
+import { InputWithIcon } from '../InputWithIcon/InputWithIcon';
+
+const dummyCategories = ['HandBags', 'Watches', 'Skincare', 'Jewllery', 'Apparels'];
+
+// eslint-disable-next-line react/prop-types, no-unused-vars
```
Comment (Rahaf-96) [inline src/Components/Header/Header.jsx]: why would you disable eslint here?

File: src/Components/Header/Header.jsx [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+export default function Header({ searchPlaceHolder, categories }) {
```
Comment (Rahaf-96) [inline src/Components/Header/Header.jsx]: please make sure to be consistent in the code syntax. 
if you want to use arrow functions syntax it should be used all over the project.
I'm noticing that you are using both the traditional function syntax declaration here and the arrow function declaration in the rest of this file.

File: src/Components/Header/Header.jsx [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+          <Box sx={{ display: { xs: 'flex'} }}>
+            <IconButton size="large" aria-label="show more" aria-haspopup="true" color="primary">
+            <img src="./icons/cart.svg" alt="Cart" />
+            </IconButton>
+          </Box>
+
+
+          <Box sx={{ display: { xs: 'flex'} }}>
+            <IconButton size="large" aria-label="show more" aria-haspopup="true" color="primary">
+              <NotificationsNoneIcon />
```
Comment (Rahaf-96) [inline src/Components/Header/Header.jsx]: is there a notification icon in desktop view? 
please note that we are using only desktop design and mobile design is removed. 
you should take the desktop design and make it responsive.

**Review by Rahaf-96 — CHANGES_REQUESTED** [review]
Great work so far! 🚀 
please check the comments...

File: public/index.html (hunk 1 above)
Comment (Mazoda) [reply public/index.html]: Done!

File: src/Components/Header/Header.jsx (hunk 2 above)
Comment (Mazoda) [reply src/Components/Header/Header.jsx]: I will create the prop-Types.
But I will disable {no-used-vars} for now until we connect with the backend

File: src/Components/Header/Header.jsx [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+
+
+        </Toolbar>
+      </AppBar>
+    </Box>
+  );
+}
+
+Header.propTypes ={
+    searchPlaceHolder: PropTypes.string.isRequired,
```
Comment (Rahaf-96) [inline src/Components/Header/Header.jsx]: please remove this prop type as u don't have this prop anymore

File: src/Components/Header/Header.jsx [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+    categories:PropTypes.array.isRequired
+}
+
+Header.defaultProps = {
+    searchPlaceHolder: 'Search Products or Brands...',
```
Comment (Rahaf-96) [inline src/Components/Header/Header.jsx]: same here

**Review by Rahaf-96 — APPROVED** [review]
Thank you for your efforts 🤩 🚀

**Pull request merged**